StaubliMovementParams params
float64 x
float64 y
float64 z
float64 rx
float64 ry
float64 rz
int32   lineCtrl
---
float64 x
float64 y
float64 z
float64 rx
float64 ry
float64 rz
---
bool error
float64 x
float64 y
float64 z
float64 rx
float64 ry
float64 rz
